src/App/Support/Traits/Tamara_Order_Trait.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tamara_Checkout\App\Support\Traits;
+
+trait Tamara_Order_Trait {
```

Comment:
Review the name, the trait should have a specific name for it's purpose, e.g. `Tamara_Order_Error_Mappings_Trait` ...